pkg/config/config.go

```diff
@@ -148,8 +154,13 @@ func (c *Config) updateComputedValues() error {
 		}
 	}
 
-	// If KAS advertise address is not configured then grab it from the service
-	// CIDR automatically.
+	// If KAS advertise address configured, we do not want to apply
+	// the IP to the internal interface.
+	if c.userSettings != nil && len(c.userSettings.ApiServer.AdvertiseAddress) != 0 {
+		c.ApiServer.SkipInterface = true
```

Comment:
If a user set `SkipInterface` to false? Shall we return an error here?

Reply:
The SkipInterface value is a computed value, so it is not currently being read from the file at all. c.userSettings.ApiServer.SkipInterface should always be false. 

@pacevedom is working on another PR that exposes the flag, although we haven't had time to talk about why that's necessary. https://github.com/openshift/microshift/pull/1478